modules/ROOT/pages/footnotes.adoc

```diff
@@ -0,0 +1,60 @@
+= Footnotes plugin
+:navtitle: Footnotes
+:description: Insert footnotes into the {productname} editor
+:keywords: footnotes, footnote, insertfootnotes, insert, footnotesupdate, mceInsertFootnote, mceUpdateFootnotes
+:pluginname: Footnotes
+:plugincode: footnotes
+:pluginminimumplan: tierthree
+
+include::partial$misc/admon-premium-plugin.adoc[]
+
+include::partial$misc/admon-requires-6.2v.adoc[]
+
+The {pluginname} plugin allows the user to insert a footnote into the {productname} editor.
+
+Each time a footnote is added, a superscripted footnote number is placed at the current insertion point, or at the end of the current selection.
+
+The plugin then adds a new line to the footnotes section located at the bottom of the editor and moves the insertion point to this new line. This allows the user to add the footnote entry related to the newly-created footnote.
```

Comment:
```suggestion
Each time a footnote is added, a superscripted footnote number is placed at the current insertion point, or at the end of the current selection.

The plugin then adds a new line to the footnotes section located at the bottom of the editor and moves the insertion point to this new line. This allows the user to instantly insert content related to the newly-created footnote in the footnote entry.
```
Should we refer to the ` footnote entry` as the location for entering content related to the respective footnote, or as the content itself within the footnotes section?

Reply:
I’ve pushed a slightly different tweaking of the sentence you re-wrote. Like your re-write it makes the distinction between a footnote entry and its contents more apparent.

It, further, doesn’t leave the inference in place that an end-user is required to add material to a footnote entry.

I don’t have an example to hand, but I can imagine a use-case that automates the creation of footnotes, or the auto-population of footnote entry contents (or both).